modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityRoleManager.cs

```diff
@@ -43,5 +44,35 @@ public async Task<IdentityRole> GetByIdAsync(Guid id)
 
             return role;
         }
+
+        public List<IdentityRole> GetDefaultRoles()
```

Comment:
We don't use IQueryable/LINQ features in the domain layer!

Remove this method. Query from repository if needed (by creating a new repo method if needed, but instead of repo.GetDefaultRoles, just name it GetList and add optional parameters to filter).